script.js

```diff
@@ -1,62 +1,101 @@
-
 const getLocationButton = document.querySelector(`#get-location`);
 const searchButton = document.querySelector(`#search-by-location`);
 const restaurantsReturned = document.querySelector(`#restaurant-returned`);
+const restaurantInfoModal = document.querySelector("#modal-restaurant-info");
+const modalToggle = document.querySelector("#open-modal");
+const modalClose = document.querySelector("#close-modal");
+
 let userLatitude = ``;
 let userLongitude = ``;
 
 getLocationButton.addEventListener(`click`, event => {
-    if (navigator.geolocation) {
-        navigator.geolocation.getCurrentPosition(function (position){
-        userLatitude = position.coords.latitude;
-        userLongitude = position.coords.longitude;
-        fetch (`https://maps.googleapis.com/maps/api/geocode/json?latlng=${userLatitude},${userLongitude}&key=AIzaSyDUU-niRiyVVeS9DtXJswEOVHjiBJevfno`)
-            .then(response => response.json())
-            .then(data =>{
-                console.log(data)
-            });
+  if (navigator.geolocation) {
```

Comment:
This has been doubled up im not sure why
can you take this chunk of code out to 21